docs/DeveloperGuide.md

```diff
@@ -257,47 +257,75 @@ _{Explain here how the data archiving feature will be implemented}_
 
 **Target user profile**:
 
-* has a need to manage a significant number of contacts
-* prefer desktop apps over other types
+d'Intérieur is designed for interior designers who:
+
+* Has a preference for CLI apps
+* has many clients and projects to keep track of
 * can type fast
 * prefers typing to mouse interactions
```

Comment:
Minor nit -- plural designers, so better to use plural form like: "prefer CLI apps", or "have many clients".